app/models/Beer.scala

```diff
@@ -15,8 +15,11 @@ abstract class Beers extends CassandraTable[Beers, Beer] with RootConnector {
   object name extends StringColumn(this) with PartitionKey
   object style extends StringColumn(this) with Index
 
-  def getByStyle(style: String): Future[List[Beer]] = {
+  def getByStyle(style: String): Future[Seq[Beer]] = {
     select.where(_.style eqs style).fetch()
   }
 
+  def all(): Future[Seq[Beer]] = {
```

Comment:
A slightly dangerous encouragement, we should maybe use `PagingState` for the standard example?

Reply:
Did not know `PagingState` was a thing.

Reply:
Cassandra has automated pagination, so you can actually do a `select.all` but in stages of 50 elements or whatever.

Reply:
Can you send me a link to docs?

Reply:
The `PagingState` should use a `QueryStringBindable` pager as defined in the scaladoc here: https://www.playframework.com/documentation/2.6.x/api/scala/index.html#play.api.mvc.QueryStringBindable

Reply:
@wsargent Yes that would make a lot of sense indeed. Apologies for the delay in replying, I have been away.